a4c-brooklyn-plugin/src/main/java/alien4cloud/brooklyn/BrooklynOrchestrator.java

```diff
@@ -39,6 +39,12 @@ public ILocationConfiguratorPlugin getConfigurator(String locationType) {
     }
 
     @Override
+    public List<PluginArchive> pluginArchives() {
+        // TODO: Implement if required (will this obviate the need to upload the normative types?)
+        return null;
```

Comment:
Without having checked how this method is used by Alien, wouldn't it be safer to return `Collections.emptyList()`?